.docker/Dockerfile-local-build

```diff
@@ -18,15 +17,14 @@ RUN go build -ldflags="-extldflags=-static" -tags sqlite,sqlite_omit_load_extens
 
 #########################
 
-FROM gcr.io/distroless/static-debian12:nonroot AS runner
+FROM gcr.io/distroless/static-debian12:debug-nonroot AS runner
 
-COPY --from=builder --chown=nonroot:nonroot /var/lib/sqlite /var/lib/sqlite
 COPY --from=builder /usr/bin/hydra /usr/bin/hydra
 
-VOLUME /var/lib/sqlite
-
 # Declare the standard ports used by hydra (4444 for public service endpoint, 4445 for admin service endpoint)
 EXPOSE 4444 4445
 
+USER nonroot
+
```

Comment:
Perhaps it's useful to keep for awareness, but you don't actually need this line as the image already defaults to the `nonroot` user.

```suggestion
```

```console
$ docker inspect gcr.io/distroless/static-debian12:debug-nonroot | jq -r '.[].Config.User'
65532
```

Extending the image like done here will retain that `USER` directive.